torch/_prims/executor.py

```diff
@@ -25,7 +30,7 @@ def execute(
     raise ValueError(msg)
 
 
-def make_traced(fn: Callable):
+def make_traced(fn: Callable[P, T]) -> Callable[P, T]:
```

Comment:
The second Callable might need to be another ParamSpec object to error on the return type if you still get errors

Reply:
If we have a second `ParamSpec` which is only used for the return type then I think there's probably no advantage over `Callable[..., T]`.

On the other hand I suspect the intention is that the GraphModule does take the same parameters as P (since it's wrapping that) so leaving it with the type: ignore is probably "correct"-ish.